core/src/main/java/org/hisp/dhis/android/core/dataelement/DataElement.java

```diff
@@ -92,6 +97,11 @@ public String categoryComboUid() {
     @Nullable
     public abstract String fieldMask();
 
+    @Nullable
+    @JsonProperty()
+    @ColumnAdapter(IgnoreAttributeValuesListAdapter.class)
+    public abstract List<AttributeValue> attributeValues();
+
     public static Builder builder() {
         return new $$AutoValue_DataElement.Builder();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.